src/backend/api/reviews.js

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import knex from '../database.js';
+
+const router = express.Router();
```

Comment:
This router should probably be exported and installed in `app.js` similar to `meals.js`. 🙂 There are some issues with the path names here as the router should be installed on `api/reviews`. Thus e.g. the first route should just be `/`.

Reply:
thank you, fixed this 👀